SEED-platform/seed
# review viewer permissions

#### What's this PR do?

#### How should this be manually tested?
As an org user with the "viewer" role, navigate to ALL pages under Inventory List and ALL pages under Property Detail.  You should be able to access all pages, but not able to do any modifying or deleting actions. You shouldn't even see the UI controls for doing those actions. This is built on top of AH so you can test a viewer leaf node user vs a root member at the same time.

![Screenshot 2024-03-04 at 3 27 55 PM](https://github.com/SEED-platform/seed/assets/2205659/43ccd05f-ef35-4c7f-92c5-48c0fc5a0ca7)

![Screenshot 2024-03-04 at 3 28 00 PM](https://github.com/SEED-platform/seed/assets/2205659/f546c072-f291-45ac-84cd-f0668cbf99b7)


#### What are the relevant tickets?
#2957
